### ISSUE TITLE
Refresh dependent views when calling `.refresh()`

### ISSUE THREAD

**Issue by radusuciu** [xelixdev/django-pgviews-redux#44]
If you have materialized views that depend on one another, they are created in the correct order, but when you call `.refresh()` the caller needs to take care to refresh all dependent views or risk working with stale data.

**Comment by mikicz** [comment]
Yes this is true. In my projects we have an utility for this, I will port it here at some point!
